docs/intro/formats.md

```diff
@@ -165,6 +173,8 @@ It is already possible to add URLs in a variety of formats that OMERO can provid
 This makes it possible to import multiple images -- even including some annotations -- to a QuPath {doc}`project <../tutorials/projects>` in one go.
 :::
 
+***TODO*** Confirm these details are all still relevant with Leos new Omero extension
+
 ### Community extensions
```

Comment:
For Updating Extensions, I would add the Add: Extensions ‣ Manage extensions ‣ Open extension directory instruction. I know its above, but if someone only clicks on Updating extansion, they won’t see it

Reply:
Noted for post I2K